ister.py

```diff
@@ -1435,7 +1443,7 @@ def main():
         install_os(args)
     except Exception as exep:
         LOG.debug("Failed: {}".format(repr(exep)))
-        # todo: Add arg for loglevel to -v
+        # TODO: Add arg for loglevel to -v
```

Comment:
Comment update should be in a different commit as well.

Reply:
And do this for all cases of TODO needing to be corrected please!

Reply:
I grepped through the code base in the repo and this was the only case.